apps/arduino_module/module_test/module_test.ino

```diff
@@ -0,0 +1,50 @@
+#include <Arduino.h>
+
+#include <Wire.h>
+#include <stdint.h>
+#include "signpost_api.h"
+
+#define ARDUINO_MODULE_I2C_ADDRESS 0x48
```

Comment:
@brghena do we have a global list of these defaults somewhere? We have the list here: https://github.com/lab11/signpost/tree/master/hardware -- should we be maintaining this or is that just unscalable?